example/IO/forest/gmsh/t8_gmsh_to_vtk.cxx

```diff
@@ -0,0 +1,131 @@
+*/
+
+/* This file provides a utility routine for quickly reading in Gmsh files and
+ * writing them out to a VTU file for visualization.
+ */
+
+#include <t8.h>                                 /* General t8code header, always include this. */
+#include <sc_options.h>                         /* CLI parser. */
+#include <t8_cmesh.h>                           /* Cmesh definition and basic interface. */
+#include <t8_forest/t8_forest_general.h>        /* Create forest. */
+#include <t8_forest/t8_forest_io.h>             /* Save forest. */
+#include <t8_schemes/t8_default/t8_default.hxx> /* Default refinement scheme. */
+#include <t8_cmesh_readmshfile.h>               /* Msh file reader. */
+
+int
+main (int argc, char **argv)
+{
+  sc_options_t *opt;   // The options we want to parse.
+  char help[BUFSIZ];   // Help message.
+  int helpme;          // Print help message.
+  int parsed;          // Return value of sc_options_parse.
+  int sreturn;         // Return value of sc functions.
+  int mpiret;          // Return value of MPI functions.
+  sc_MPI_Comm comm;    // The MPI communicator.
+  t8_cmesh_t cmesh;    // The cmesh we read in from the msh file.
+  t8_forest_t forest;  // The forest we want to refine.
+
+  const char *fileprefix = NULL;  // The prefix of the msh (and brep) file.
+  int level;                      // Uniform refinement level of the forest.
+  int dim;                        // The dimension of the mesh.
+  int use_cad;                    // Use CAD-based curvilinear geometry.
+
+  // Help message.
+  sreturn = snprintf (help, BUFSIZ, "Read in a `.msh` file generated by Gmsh and write it into a VTU file.\n");
+
+  if (sreturn >= BUFSIZ) {
+    // The help message was truncated.
+    // Note: gcc >= 7.1 prints a warning if we
+    // do not check the return value of snprintf.
+    t8_debugf ("Warning: Truncated help message to '%s'\n", help);
+  }
+
+  // Initialize MPI. This has to happen before we initialize sc or t8code.
+  mpiret = sc_MPI_Init (&argc, &argv);
+  // Error check the MPI return value.
+  SC_CHECK_MPI (mpiret);
+
+  // Initialize the sc library, has to happen before we initialize t8code.
+  sc_init (sc_MPI_COMM_WORLD, 1, 1, NULL, SC_LP_ESSENTIAL);
+  // Initialize t8code with log level SC_LP_PRODUCTION. See sc.h for more info on the log levels.
+  t8_init (SC_LP_ESSENTIAL);
+
+  // We will use MPI_COMM_WORLD as a communicator.
+  comm = sc_MPI_COMM_WORLD;
+
+  // Initialize command line argument parser.
+  opt = sc_options_new (argv[0]);
+  sc_options_add_switch (opt, 'h', "help", &helpme, "Display a short help message.");
+  sc_options_add_string (opt, 'f', "fileprefix", &fileprefix, NULL, "Fileprefix of the msh and brep files.");
+  sc_options_add_int (opt, 'l', "level", &level, 2, "The uniform refinement level. Default: 2");
+  sc_options_add_int (opt, 'd', "dimension", &dim, 3, "The dimension of the mesh. Default: 3");
+  sc_options_add_int (
+    opt, 'o', "use_cad", &use_cad, 0,
```

Comment:
lets use c for cad instead of o. The o is a remnant from the occ->cad rename. Maybe we can also change this in the naca tutorial

```suggestion
    opt, co', "use_cad", &use_cad, 0,
```